hedyorg/hedy
# 📜 Make example code look more like a 'T'

**How to test**

Follow these steps to verify this PR works as intended:

* Go to level 2
* adventure Turtle
* Copy example code of first assignment 
* Run the example code en see a T
